content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,66 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` is deprecated due to a [bug](https://github.com/istio/istio/issues/8825). You should
+migrate to `ClusterRbacConfig` if you are currently using `RbacConfig`. The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
+
+To migrate the `RbacConfig` to `ClusterRbacConfig`:
+
+1. We provide the `convert_RbacConfig_to_ClusterRbacConfig.sh` script to automate the migration,
+   depending on if you have the [Istio installation package](/docs/setup/kubernetes/download-release):
```

Comment:
I don't see how this dependence is important. IMO, mentioning that it is included in the installation package is enough to cover that situation. I would get rid of lines 271-279 and just provide the users with the command for direct download and execution. If you follow said advice, the text needs to change to:
To automate the migration, we developed the `convert_RbacConfig_to_ClusterRbacConfig.sh` script. The script is included in the [Istio installation package](/docs/setup/kubernetes/download-release). 
Download and run the script with the following command:
<command on lines 283 -285>

Reply:
The Istio installation package is actually optional, user could install Istio without using the Istio installation package. That's why I provide two options here: if the user installed Istio via the installation package, they can find the script already there.
If the user installs it in some custom way, they can only download the script (not the Istio installation package) nad use the script directly .

Reply:
I understand, but my point is: We don't need to tell the readers how to get the installation package here. That bit of information is given when they install. It is enough to mention they can find the script there if they already have the package, which my suggestions do.